controllers/providers/aws/ssm.go

```diff
@@ -0,0 +1,71 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/request"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ssm"
+	"github.com/aws/aws-sdk-go/service/ssm/ssmiface"
+	"github.com/keikoproj/aws-sdk-go-cache/cache"
+	"github.com/keikoproj/instance-manager/controllers/common"
+)
+
+type architectureMap map[string]string
+
+const (
+	EksOptimisedAmiPath           = "/aws/service/eks/optimized-ami/%s/amazon-linux-2/recommended/image_id"
+	EksOptimisedAmazonLinux2ARM64 = "/aws/service/eks/optimized-ami/%s/amazon-linux-2-arm64/recommended/image_id"
+	EksOptimisedBottlerocket      = "/aws/service/bottlerocket/aws-k8s-%s/x86_64/latest/image_id"
+	EksOptimisedWindowsCore       = "/aws/service/ami-windows-latest/Windows_Server-2019-English-Core-EKS_Optimized-%s/image_id"
+	EksOptimisedWindowsFull       = "/aws/service/ami-windows-latest/Windows_Server-2019-English-Full-EKS_Optimized-%s/image_id"
+)
+
+var (
+	EksAmis = map[string]architectureMap{
+		"amazonlinux2": architectureMap{
+			"x86_64": EksOptimisedAmiPath,
+			"arm64":  EksOptimisedAmazonLinux2ARM64,
+		},
+		"bottlerocket": architectureMap{
+			"x86_64": EksOptimisedBottlerocket,
```

Comment:
BottleRocket should have an arm64 ami as well

Reply:
Following https://docs.aws.amazon.com/eks/latest/userguide/retrieve-ami-id-bottlerocket.html Amazon only document a x86_64 AMI; I've tried to follow the expected path and got a result - fingers crossed continues to be supported :)

```
{
    "Parameter": {
        "Name": "/aws/service/bottlerocket/aws-k8s-1.19/arm64/latest/image_id",
        "Type": "String",
        "Value": "ami-09bfe0d7ac581d319",
        "Version": 10,
        "LastModifiedDate": "2021-08-07T03:26:47.119000+10:00",
        "ARN": "arn:aws:ssm:us-west-2::parameter/aws/service/bottlerocket/aws-k8s-1.19/arm64/latest/image_id",
        "DataType": "text"
    }
}
```